creator-node/src/services/initAudiusLibs.js

```diff
@@ -8,7 +8,11 @@ const { logger: genericLogger } = require('../logging')
  *
  * Configures dataWeb3 to be internal to libs, logged in with delegatePrivateKey in order to write chain TX
  */
-module.exports = async (excludeDiscovery = false, logger = genericLogger) => {
+module.exports = async (
+  contractsOnly = false,
```

Comment:
this pattern of ..Only and exclude... seems a bit confusing. my personal take would be to have enableContracts, enableDiscovery etc and just default the values to true and override in the right places

Reply:
I agree it felt a bit messy writing it. just made your requested change now